scripts/e2e-test.sh

```diff
@@ -47,6 +47,7 @@ setup_executor () {
     echo "$port"
     docker inspect "$containerName" | jq .
     docker exec "$EXEC_CONTAINER_NAME" sh -c "sed -i 's/https:\/\/localhost:[0-9]\+/https:\/\/localhost:6443/g' /.kube/config"
+    docker exec "$EXEC_CONTAINER_NAME" sh -c "sed -i 's/https:\/\/127.0.0.1:[0-9]\+/https:\/\/127.0.0.1:6443/g' /.kube/config"
```

Comment:
Can you explain the purpose of adding this (or why it was necessary)?

Reply:
I had to add this for my local testing. Docker appeared to use the IP instead of `localhost` for me. I figured it couldn't hurt to have it here, but I can remove it if we'd rather keep it minimal.

Reply:
Nope, seems fine to keep it. Just wanted to check.